### DISCUSSION TITLE
Getting error while running sh script

### DISCUSSION THREAD

**Discussion by GoutamVerma** [ACINQ/eclair#2267] [Node operators]
Hello Eclair Team,

I am getting an error while running sh command. `
eclair-node-<version>-<commit_id>/bin/eclair-node.sh
`
Error: `fatal error: could not connect to bitcoind using json-rpc
`

First I have downloaded and configured bitcoin core and JDK 11.
Then downloaded the latest release of eclair 0.7.0.

Here is my eclair.conf
```
eclair.node-alias=eclair
eclair.node-color=49daaa
eclair.server.port=9735
eclair.api.port=8080
eclair.bitcoind.rpcuser=foo
eclair.bitcoind.rpcpassword=bar
eclair.bitcoind.zmqblock="tcp://127.0.0.1:29000"
eclair.server.public-ips=["49.36.25.1"]
```
And bitcoin.conf
```
testnet=1
server=1
rpcuser=foo
rpcpassword=bar
txindex=1
zmqpubhashblock=tcp://127.0.0.1:29000
zmqpubrawtx=tcp://127.0.0.1:29000
```

**Comment by t-bast** [comment]
I have a couple of comments:

1. Are you sure you want to use `testnet`? It would be much easier for you to use `regtest` instead (in `bitcoin.conf` set `regtest=1`).
2. You're missing `eclair.bitcoind.zmqtx="tcp://127.0.0.1:29001"` in your `eclair.conf`
3. You're missing `eclair.api.enabled=true` in your `eclair.conf`
4. You're not setting `eclair.bitcoin.rpcport`

You should have a look at https://github.com/t-bast/lightning-cfg/
This should help you get started running a `bitcoind` node and an `eclair` node on `regtest`.

You can find information about bitcoin's `regtest` mode here: https://developer.bitcoin.org/examples/testing.html

**Comment by GoutamVerma** [comment]
I have added these parameters but it does not work😣 and shows same error again. I installed the latest version of bitcoin core maybe that's the problem.

And I decided to work on Windows and am done with the bitcoin core setup only the eclair setup is left. 
For setting up with the eclair I am pretty much sure about how to launch it on Windows (through code, exe, or anything else)

For the coding work, I am ready with IntelliJ IDEA and plugins for Scala.

**Reply by t-bast** [reply]
It has nothing to do with the version of bitcoin core in my opinion.
Take your time to read the documentation I linked (and what I sent you on Discord), this should contain everything you need.

**Reply by GoutamVerma** [reply]
Okay Thank you!
